operators/go/gen_subm_operator.sh

```diff
@@ -13,8 +13,8 @@ add_routeagent=true
 openapi_checks_enabled=false
 push_image=false
 op_dir=$GOPATH/src/github.com/submariner-operator/submariner-operator
-op_gen_dir=$GOPATH/src/github.com/submariner-io/submariner/operators/go
-op_out_dir=$GOPATH/src/github.com/submariner-io/submariner/operators/go/submariner-operator
+op_gen_dir=$(pwd)
```

Comment:
Can you help me understand the motivation for this one? It definitely makes sense for script portability, but I think there are advantages to keeping everything relative to GOPATH.

Also, I worry that this would encourage folks to do development work outside of their go/src/ directory, which I thinks breaks assumptions of other tooling.

Reply:
Update: I went ahead and made these changes, not worth nit-picking over. :+1: